<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartingBonus</class>
 <widget class="QDialog" name="StartingBonus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scenario Properties</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelSelBonusType">
     <property name="text">
      <string>Select a bonus type</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioButtonSpell">
     <property name="text">
      <string>Spell</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioButtonCreature">
     <property name="text">
      <string>Creature</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioButtonBuilding">
     <property name="text">
      <string>Building</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioButtonArtifact">
     <property name="text">
      <string>Artifact</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioButtonSpellScroll">
     <property name="text">
      <string>Spell scroll</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioButtonPrimarySkill">
     <property name="text">
      <string>Primary skill</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioButtonSecondarySkill">
     <property name="text">
      <string>Secondary skill</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioButtonResource">
     <property name="text">
      <string>Resource</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabSpell">
      <attribute name="title">
       <string>Spell</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="labelSpellRecipient">
         <property name="text">
          <string>Recipient</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxSpellRecipient"/>
       </item>
       <item>
        <widget class="QLabel" name="labelSpellSpell">
         <property name="text">
          <string>Spell</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxSpellSpell"/>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabCreature">
      <attribute name="title">
       <string>Creature</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="labelCreatureRecipient">
         <property name="text">
          <string>Recipient</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxCreatureRecipient"/>
       </item>
       <item>
        <widget class="QLabel" name="labelCreatureCreatureType">
         <property name="text">
          <string>Creature type</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxCreatureCreatureType"/>
       </item>
       <item>
        <widget class="QLabel" name="labelCreatureQuantity">
         <property name="text">
          <string>Quantity</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxCreatureQuantity"/>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabBuilding">
      <attribute name="title">
       <string>Building</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="labelBuildingBuilding">
         <property name="text">
          <string>Building</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxBuildingBuilding"/>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabArtifact">
      <attribute name="title">
       <string>Artifact</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="labelArtifactRecipient">
         <property name="text">
          <string>Recipient</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxArtifactRecipient"/>
       </item>
       <item>
        <widget class="QLabel" name="labelArtifactArtifact">
         <property name="text">
          <string>Artifact</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxArtifactArtifact"/>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabSpellScroll">
      <attribute name="title">
       <string>Spell scroll</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="labelSpellScrollRecipient">
         <property name="text">
          <string>Recipient</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxSpellScrollRecipient"/>
       </item>
       <item>
        <widget class="QLabel" name="labelSpellScrollSpell">
         <property name="text">
          <string>Spell</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxSpellScrollSpell"/>
       </item>
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPrimarySkill">
      <attribute name="title">
       <string>Primary skill</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QLabel" name="labelPrimarySkillRecipient">
         <property name="text">
          <string>Recipient</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxPrimarySkillRecipient"/>
       </item>
       <item>
        <widget class="QLabel" name="labelPrimarySkillAttack">
         <property name="text">
          <string>Attack skill</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxPrimarySkillAttack"/>
       </item>
       <item>
        <widget class="QLabel" name="labelPrimarySkillDefense">
         <property name="text">
          <string>Defense skill</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxPrimarySkillDefense"/>
       </item>
       <item>
        <widget class="QLabel" name="labelPrimarySkillSpell">
         <property name="text">
          <string>Spell power</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxPrimarySkillSpell"/>
       </item>
       <item>
        <widget class="QLabel" name="labelPrimarySkillKnowledge">
         <property name="text">
          <string>Knowledge</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxPrimarySkillKnowledge"/>
       </item>
       <item>
        <spacer name="verticalSpacer_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabSecondarySkill">
      <attribute name="title">
       <string>Secondary skill</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QLabel" name="labelSecondarySkillRecipient">
         <property name="text">
          <string>Recipient</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxSecondarySkillRecipient"/>
       </item>
       <item>
        <widget class="QLabel" name="labelSecondarySkillSecondarySkill">
         <property name="text">
          <string>Secondary skill</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxSecondarySkillSecondarySkill"/>
       </item>
       <item>
        <widget class="QLabel" name="labelSecondarySkillMastery">
         <property name="text">
          <string>Mastery</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxSecondarySkillMastery"/>
       </item>
       <item>
        <spacer name="verticalSpacer_7">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabResource">
      <attribute name="title">
       <string>Resource</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QLabel" name="labelResourceResourceType">
         <property name="text">
          <string>Resource type</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxResourceResourceType"/>
       </item>
       <item>
        <widget class="QLabel" name="labelResourceQuantity">
         <property name="text">
          <string>Quantity</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxResourceQuantity"/>
       </item>
       <item>
        <spacer name="verticalSpacer_8">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
